### PR TITLE
Vpc route info

### PR DESCRIPTION
**Pull request by Polina-Gubina**

### CODE REVIEW

**Comment by Polina-Gubina** [comment]
recheck

File: plugins/modules/vpc_route_info.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+version_added: "0.0.2"
+author: "Polina Gubina (@polina-gubina)"
+description:
+  - Get a generator of vpc routes info from the OTC.
+options:
+  id:
+    description:
+      - Route ID.
+    type: str
+  tenant_id:
```
Comment (gtema) [inline plugins/modules/vpc_route_info.py:28]: this should be project_id

File: plugins/modules/vpc_route_info.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    description:
+      - Tenant id.
+    type: str
+  vpc_id:
```
Comment (gtema) [inline plugins/modules/vpc_route_info.py:32]: router_id

File: plugins/modules/vpc_route_info.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      sample: "192.168.200.0/24"
+    nexthop:
+      description: The next hop. If type is peering, it is the VPC peering connection ID
+      type: str
+      sample: "7375f1cd-6fe1-4d47-8888-c5c5a64298d8"
+    type:
+      description: The route type.
+      type: str
+      sample: "peering"
+    vpc_id:
```
Comment (gtema) [inline plugins/modules/vpc_route_info.py:70]: I think those are returned as router_id and project_id as well

File: plugins/modules/vpc_route_info.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        vpc_id_filter = self.params['vpc_id']
+        destination_filter = self.params['destination']
+        type_filter = self.params['type']
+
+        data = []
+        query = {}
+        if id_filter:
+            query['id'] = id_filter
+        if tenant_id_filter:
+            query['status'] = tenant_id_filter
```
Comment (gtema) [inline plugins/modules/vpc_route_info.py:113]: I guess there is typo here - 'status' is definitely wrong

File: plugins/modules/vpc_route_info.py [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+        if vpc_id_filter:
+            query['project_id'] = vpc_id_filter
```
Comment (gtema) [inline plugins/modules/vpc_route_info.py:115]: here also mixed

**Review by gtema — DISMISSED** [review]

**Comment by Polina-Gubina** [comment]
recheck

**Comment by Polina-Gubina** [comment]
recheck

**Comment by Polina-Gubina** [comment]
recheck

**Comment by Polina-Gubina** [comment]
do not do recheck

**Comment by Polina-Gubina** [comment]
recheck

**Comment by Polina-Gubina** [comment]
recheck

File: tests/integration/targets/vpc_route_info/tasks/main.yaml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    - name: Debug vpc route
+      debug:
+        var: vpc_routes
+
+    - name: assert result
+      assert:
+        that:
+          - vpc_routes is success
+          - vpc_routes is not changed
+          - vpc_routes.vpc_routes is defined
```
Comment (anton-sidelnikov) [inline tests/integration/targets/vpc_route_info/tasks/main.yaml:16]: endline

**Review by anton-sidelnikov — APPROVED** [review]

File: plugins/modules/vpc_route_info.py [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+---
+module: vpc_route_info
+short_description: Get information about vpc routes info
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.0.2"
```
Comment (tischrei) [inline plugins/modules/vpc_route_info.py:19]: 0.2.0 is it is mentioned in our tag

**Review by tischrei — CHANGES_REQUESTED** [review]

**Review by tischrei — APPROVED** [review]

**Pull request merged**